### ISSUE TITLE
4.11: builds are broken

### ISSUE THREAD

**Issue by cgwalters** [openshift/os#843]
```
 Enabled rpm-md repositories: rhel-8-baseos rhel-8-appstream rhel-8-server-ose rhel-8-fast-datapath
Updating metadata for 'rhel-8-baseos'...done
Updating metadata for 'rhel-8-appstream'...done
Updating metadata for 'rhel-8-server-ose'...done
Updating metadata for 'rhel-8-fast-datapath'...done
Importing rpm-md...done
rpm-md repo 'rhel-8-baseos'; generated: 2022-05-04T07:23:27Z solvables: 11158
rpm-md repo 'rhel-8-appstream'; generated: 2022-06-07T09:36:05Z solvables: 24626
rpm-md repo 'rhel-8-server-ose'; generated: 2022-06-09T21:26:03Z solvables: 940
rpm-md repo 'rhel-8-fast-datapath'; generated: 2022-05-27T18:05:26Z solvables: 793
Resolving dependencies...done
error: Could not depsolve transaction; 4 problems detected:
 Problem 1: conflicting requests
  - nothing provides libsubid.so.3()(64bit) needed by podman-2:4.0.2-4.rhaos4.11.el8.x86_64
 Problem 2: conflicting requests
  - nothing provides libsubid.so.3()(64bit) needed by podman-2:4.0.2-4.rhaos4.11.el8.x86_64
 Problem 3: conflicting requests
  - package toolbox-0.0.9-1.rhaos4.11.el8.noarch requires podman, but none of the providers can be installed
  - package podman-1.0.0-2.git921f98f.module+el8+2785+ff8a053f.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-10.module+el8.2.0+6064+4ca4827e.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.4.0+11310+8c67a752.x86_64 is filtered out by modular filtering
  - package podman-2.0.5-5.module+el8.3.0+8221+97165c3f.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-7.module+el8.4.0+11311+9da8acfb.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-26.module+el8.4.0+10193+e90fd8eb.x86_64 is filtered out by modular filtering
  - package podman-1.0.5-1.gitf604175.module+el8.0.0+4017+bbba319f.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-11.module+el8.2.0+6369+1f4293b4.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-4.git921f98f.module+el8.2.0+6370+6fb6c8ca.x86_64 is filtered out by modular filtering
  - package podman-1.4.2-5.module+el8.1.0+4240+893c1ab8.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-24.module+el8.3.0+9348+d780f094.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-6.git921f98f.module+el8.3.0+8236+8e428216.x86_64 is filtered out by modular filtering
  - package podman-1.0.3-1.git9d78c0c.module+el8.0.0.z+3717+fdd07b7c.x86_64 is filtered out by modular filtering
  - package podman-2.2.1-7.module+el8.3.1+9857+68fb1526.x86_64 is filtered out by modular filtering
  - package podman-3.2.3-0.11.module+el8.4.0+12050+ef972f71.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-3.git921f98f.module+el8.1.0+4241+a7060183.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.4.0+10614+dd38312c.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-4.git921f98f.module+el8.1.0+4908+72a45cef.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8.0.0+4014+8662b6b2.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-10.module+el8.2.0+6063+e761893a.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8.0.0+2958+4e823551.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.4.0+10607+f4da7515.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-11.module+el8.2.0+6368+cf16aa14.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8.0.0+2956+30df4692.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-23.module+el8.3.0+8233+627fbb78.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-26.module+el8.3.0+10188+4c10031c.x86_64 is filtered out by modular filtering
  - package podman-1.9.3-2.module+el8.2.1+6867+366c07d6.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8+2784+9a0c1dfe.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-23.module+el8.3.0+8377+eff33c85.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-12.module+el8.2.0+6670+014d0ff8.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-15.module+el8.2.0+7290+954fb593.x86_64 is filtered out by modular filtering
  - package podman-3.2.3-0.10.module+el8.4.0+11989+6676f7ad.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-4.module+el8.1.1+5885+44006e55.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-12.module+el8.2.0+6669+dde598ec.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-2.module+el8.1.1+5363+bf8ff1af.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-8.git921f98f.module+el8.3.0+10171+12421f43.x86_64 is filtered out by modular filtering
  - package podman-1.4.2-6.module+el8.1.0+4830+f49150d7.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-26.module+el8.5.0+10223+f7559c42.x86_64 is filtered out by modular filtering
  - package podman-3.3.1-9.module+el8.5.0+12697+018f24d7.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.5.0+12609+beaa716d.x86_64 is filtered out by modular filtering
  - package podman-1:3.4.2-9.module+el8.5.0+13852+150547f7.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-7.module+el8.5.0+12960+5b7d72b5.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-7.module+el8.4.0+12906+0cce23da.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-8.module+el8.5.0+14784+8c48f0fc.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-28.module+el8.5.0+14796+e4386a39.x86_64 is filtered out by modular filtering
  - nothing provides libsubid.so.3()(64bit) needed by podman-2:4.0.2-4.rhaos4.11.el8.x86_64
 Problem 4: conflicting requests
  - package toolbox-0.0.9-1.rhaos4.11.el8.noarch requires podman, but none of the providers can be installed
  - package podman-1.0.0-2.git921f98f.module+el8+2785+ff8a053f.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-10.module+el8.2.0+6064+4ca4827e.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.4.0+11310+8c67a752.x86_64 is filtered out by modular filtering
  - package podman-2.0.5-5.module+el8.3.0+8221+97165c3f.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-7.module+el8.4.0+11311+9da8acfb.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-26.module+el8.4.0+10193+e90fd8eb.x86_64 is filtered out by modular filtering
  - package podman-1.0.5-1.gitf604175.module+el8.0.0+4017+bbba319f.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-11.module+el8.2.0+6369+1f4293b4.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-4.git921f98f.module+el8.2.0+6370+6fb6c8ca.x86_64 is filtered out by modular filtering
  - package podman-1.4.2-5.module+el8.1.0+4240+893c1ab8.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-24.module+el8.3.0+9348+d780f094.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-6.git921f98f.module+el8.3.0+8236+8e428216.x86_64 is filtered out by modular filtering
  - package podman-1.0.3-1.git9d78c0c.module+el8.0.0.z+3717+fdd07b7c.x86_64 is filtered out by modular filtering
  - package podman-2.2.1-7.module+el8.3.1+9857+68fb1526.x86_64 is filtered out by modular filtering
  - package podman-3.2.3-0.11.module+el8.4.0+12050+ef972f71.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-3.git921f98f.module+el8.1.0+4241+a7060183.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.4.0+10614+dd38312c.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-4.git921f98f.module+el8.1.0+4908+72a45cef.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8.0.0+4014+8662b6b2.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-10.module+el8.2.0+6063+e761893a.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8.0.0+2958+4e823551.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.4.0+10607+f4da7515.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-11.module+el8.2.0+6368+cf16aa14.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8.0.0+2956+30df4692.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-23.module+el8.3.0+8233+627fbb78.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-26.module+el8.3.0+10188+4c10031c.x86_64 is filtered out by modular filtering
  - package podman-1.9.3-2.module+el8.2.1+6867+366c07d6.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-2.git921f98f.module+el8+2784+9a0c1dfe.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-23.module+el8.3.0+8377+eff33c85.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-12.module+el8.2.0+6670+014d0ff8.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-15.module+el8.2.0+7290+954fb593.x86_64 is filtered out by modular filtering
  - package podman-3.2.3-0.10.module+el8.4.0+11989+6676f7ad.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-4.module+el8.1.1+5885+44006e55.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-12.module+el8.2.0+6669+dde598ec.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-2.module+el8.1.1+5363+bf8ff1af.x86_64 is filtered out by modular filtering
  - package podman-1.0.0-8.git921f98f.module+el8.3.0+10171+12421f43.x86_64 is filtered out by modular filtering
  - package podman-1.4.2-6.module+el8.1.0+4830+f49150d7.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-26.module+el8.5.0+10223+f7559c42.x86_64 is filtered out by modular filtering
  - package podman-3.3.1-9.module+el8.5.0+12697+018f24d7.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-6.module+el8.5.0+12609+beaa716d.x86_64 is filtered out by modular filtering
  - package podman-1:3.4.2-9.module+el8.5.0+13852+150547f7.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-7.module+el8.5.0+12960+5b7d72b5.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-7.module+el8.4.0+12906+0cce23da.x86_64 is filtered out by modular filtering
  - package podman-3.0.1-8.module+el8.5.0+14784+8c48f0fc.x86_64 is filtered out by modular filtering
  - package podman-1.6.4-28.module+el8.5.0+14796+e4386a39.x86_64 is filtered out by modular filtering
  - nothing provides libsubid.so.3()(64bit) needed by podman-2:4.0.2-4.rhaos4.11.el8.x86_64
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-06-09T22:28:00Z"} 
```

I think we must be missing some bit of the CI cleanup that happened on master to fix up the repository configs?

**Comment by miabbott** [comment]
Strangely the downstream builds are happily moving along, so there is a drift between the CI config and downstream

**Comment by travier** [comment]
Closed by #846
/close
